.github/actions/change-workflow-instance-states/action.yml

```diff
@@ -21,4 +21,4 @@ runs:
           aws-secret-access-key: ${{ env.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ env.AWS_DEFAULT_REGION }}
       - run: .github/scripts/change-workflow-instance-states.py ${{ inputs.platform }} ${{ github.run_id }} ${{ inputs.new-state }} ${{ inputs.github-token }}
-        shell: bash
+        shell: bash -el {0}
```

Comment:
what is this change doing? Is this is separate to the conda req changes?

Reply:
This change is needed so that each shell sources the `.bashrc` that auto-activates the CI environment (given by the conda install GH action).